### ISSUE TITLE
New Sysmon Parser

### ISSUE THREAD

**Issue by strengthnotes** [defensivedepth/Sysmon_ELSA_Parsers#6]
Josh, Was just checking to see if you had a chance to write ELSA parsers for the rest of the sysmon events as you mentioned here?

https://groups.google.com/forum/#!searchin/security-onion/sysmon%7Csort:date/security-onion/-CUWH4rpIcs/NOOteJWCBgAJ

I just wanted to make sure as I am starting to finish off the rest of the event parsers for ELSA and don't want to duplicate the effort.

https://github.com/jtaylo78/Sysmon_ELSA_Parsers/blob/Sysmon-4/sysmon.sql
https://github.com/jtaylo78/Sysmon_ELSA_Parsers/blob/master/sysmon4

**Comment by defensivedepth** [comment]
No, I have not. The new stuff I am preparing for the SO Conference has taken up my spare time...

Are the parsers you linked to complete?

**Comment by strengthnotes** [comment]
Great, Just wanted to make sure I didn't miss something and was duplicate effort. 

These parsers are not complete at this time.  If life cooperates should be done this weekend or next.

Look forward to seeing the new stuff you are prepping.

Thanks
